index.js

```diff
@@ -1,3 +1,9 @@
+var assign = require('object-assign');
```

Comment:
Was a bit sceptical about adding a module for this, because until now this module has had no dependencies. I'd have to add a bundler, which is fine, but will add a bit of overhead. It's also missing from the `package.json`